frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/Delete_HPA_Action.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Delete HPA action
+	As a user, I want to remove the HPA assigned to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment/deployment-config 
```

Comment:
Keep it either deployment or deployment config. While writing step definition, we need to update it again. It's better to update it now.